### PR TITLE
[HDRP][Path Tracing] Updated What's new 12 file

### PR DESCRIPTION
**Pull request by eturquin**
Mentioned support for AxF and Stacklit materials in "What's new in version 12" file (in addition to Fabric).
Also updated the Features page with a path traced materials section.

### CODE REVIEW

**Review by sebastienlagarde — COMMENTED** [review]
Please also update the feature list: https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@12.0/manual/HDRP-Features.html thanks

**Comment by eturquin** [comment]
> Please also update the feature list: https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@12.0/manual/HDRP-Features.html thanks

Done; I also added a banner-style picture.

File: com.unity.render-pipelines.high-definition/Documentation~/HDRP-Features.md [hunk 1]
```diff
@@ -386,6 +386,12 @@ Path tracing now supports fog absorption. Like SSS, to use this feature, enable
 
 For information on fog in HDRP, see [fog](Override-Fog.md).
 
+#### Materials support
+
+![](Images/Path-traced-materials-Feature.png)
+
+Path tracing supports almost all the rasterization engine [materials](#Material) listed above: it can render Lit, Layered Lit, Unlit, Stacklit, Fabric and AxF materials. Note that objects using non-HDRP materials simply won't appear in the image, when in path tracing mode.
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/HDRP-Features.md:393]: Path tracing can render the following rasterization engine [Materials](#Material): 

- Lit
- Layered Lit
- Unlit,
- Stacklit
- Fabric
- AxF 

When path tracing is enabled, objects that use non-HDRP Materials won't appear in the final image.

File: com.unity.render-pipelines.high-definition/Documentation~/whats-new-12.md [hunk 2]
```diff
@@ -70,11 +70,14 @@ From HDRP 12.0, The AOV API includes the following improvements:
...
-### Path traced fabric material
+### More path traced materials
 
 ![](Images/HDRPFeatures-FabricPT.png)
 
-HDRP's path tracer now offers support for the fabric material, in both its cotton/wool and silk variants.
+HDRP's path tracer now offers support for additional materials:
+- Fabric, in both its cotton/wool and silk variants.
+- AxF, in both its SVBRDF and car paint variants.
+- Stacklit.
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/whats-new-12.md:81]: HDRP path tracing now supports the following Materials:
- Fabric: Cotton/wool and silk variants.
- AxF: SVBRDF and car paint variants.
- Stacklit.

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
Suggested some changes.

**Review by sebastienlagarde — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
As Vic's feedback have been adress, merging

**Pull request merged**